### ISSUE TITLE
Make slack notifications optional

### ISSUE THREAD

**Issue by LucasNova** [life-exe/devops_ue#3]
Your game-target pipeline failed when you didn't configure `slack notifications` in Jenkins. Ideally, give developers a choice between (None, Slack, Discord, Telegram, etc).

**Comment by life-exe** [comment]
Fair enough. The first step could be to add a conditional step for slack notifications.
